### PR TITLE
Allow variable keys, times, and clefs across different measures in lilypond

### PR DESCRIPTION
**Pull request by evanmcallaway**
Curently only supporting alto, tenor, and bass clef.  Also parse some more nodes into music XML that aren't being expressed yet.

### CODE REVIEW

File: lib/musicxml/node/base.rb [hunk 1]
```diff
@@ -23,7 +23,12 @@ def parse_nodes(node)
           self.class.stored_nodes.each do |name|
             klass = find_class(symbol_to_class(name))
             node_list = node.search(symbol_to_node(name)).map do |child_node|
-              klass.new(child_node)
+              begin
+                klass.new(child_node)
+              rescue => exception
+                puts "Could not create node of type #{klass}: #{exception.message}"
```
Comment (kddnewton) [inline lib/musicxml/node/base.rb]: Hesitant to put in a global rescue here, do we need it?

Comment (evanmcallaway) [inline lib/musicxml/node/base.rb]: No not strictly necessary, but useful for debugging as it gives a more meaningful error message than a full trace.  Perhaps we should re-throw after printing the class and node that are behaving naughty?

**Comment by kddnewton** [comment]
Just pushed the singular and plural attributes, and some more vexflow conversions. Can you rebase this branch against master?

**Comment by evanmcallaway** [comment]
Okay, rebased and seems to be working with supported lilypond attributes.

**Pull request merged**